knowage-core/src/main/java/it/eng/spagobi/api/AbstractDocumentResource.java

```diff
@@ -257,7 +257,7 @@ public Response deleteCurrentTemplate(String documentLabel) {
 		return Response.ok().build();
 	}
 
-	public Response deleteTemplateById(String documentLabel, Integer templateId) {
+	protected Response deleteTemplateById(String documentLabel, Integer templateId) {
```

Comment:
If possible, I would prefer private but this is an abstract class, we need a check.

Reply:
This one can't be private, it is being used in DocumentResource.java.